homeassistant/config.py

```diff
@@ -167,7 +167,7 @@ def _filter_bad_internal_external_urls(conf: dict) -> dict:
             # We warn but do not fix, because if this was incorrectly configured,
             # adjusting this value might impact security.
             _LOGGER.warning(
-                "Invalid %s set. It's not allowed to have a path (/bla)", key
+                f"Invalid {key} set. It's not allowed to have a path (/bla)"
```

Comment:
You should never f-string log strings in Python. In case of logging, they are evaluated lazy.

Reply:
@frenck, thanks for telling me! I will try and resolve this issue ASAP.

Reply:
@frenck this is resolved now.